flows/inference.py

```diff
@@ -393,7 +399,7 @@ async def run_classifier_inference_on_document(
         store_labels(
             config=config,
             labels=doc_labels,
-            file_stem=file_stem,
+            file_stem=remove_translated_suffix(file_stem),
```

Comment:
@mpjuhasz has the concern here that we lose track of the source document.

Reply:
Idea that we could update the indexer to accept _translated_en documents and index these by stripping the suffix.

Reply:
If we did want to add a check to the indexer that the document had the correct language we could load it from s3.

Reply:
OR update the labelled passages object.

Reply:
OR prune and trust the above logic.